src/testing/testing.ts

```diff
@@ -126,7 +132,7 @@ export const createTesting = async (config: ValidatedConfig): Promise<Testing> =
 
           const styleUrl = getAppStyleUrl(config, devServer.browserUrl);
           if (styleUrl) {
-            env.__STENCIL_APP_STYLE_URL__ = getAppStyleUrl(config, devServer.browserUrl);
+            env.__STENCIL_APP_STYLE_URL__ = styleUrl;
```

Comment:
I have no idea why we had an extraneous call to `getAppStyleUrl` here - I can't find any side effects as a result of calling it, so my best guess is that it was just an oversight at author time/at some point if there was a side effect.